### PR TITLE
Feature/#8384 export certificates/#9062 action sheet

### PR DESCRIPTION
**Pull request by pascal-brause**
## Description
In a HealthCertificate details, this replaces the "remove" button by a "more" button. Tapping this will open an action sheet with the possibility to remove the certificate, show a printable version or to cancel the action sheet.

This PR adds the action sheet and moves the remove certificate functionality to the action sheets button. The remaining functionality comes in the next PR.

## Link to Jira
Main Story: https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8384
Subtask: https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9062

## Screenshots
9062-more
9062-more-dark
9062-actionsheet
9062-actionsheet-dark

### CODE REVIEW

**Review by KaiTeuber — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewController.swift [hunk 1]
```diff
@@ -15,11 +15,11 @@ class HealthCertificateViewController: UIViewController, UITableViewDataSource,
 		vaccinationValueSetsProvider: VaccinationValueSetsProviding,
 		dismiss: @escaping () -> Void,
 		didTapValidationButton: @escaping () -> Void,
-		didTapDeleteButton: @escaping () -> Void
+		didTapActionSheet: @escaping () -> Void
```
Comment (marcussc) [inline src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewController.swift:18]:
```suggestion
		didMoreButton: @escaping () -> Void
```

Just an Idea that would not hint that there is an ActionSheet that has to be used.

**Review by marcussc — APPROVED** [review]
I like that this "small" task is its own PR.
Otherwise just a small naming thing/idea.

**Review by NataliaLemmerth — APPROVED** [review]
UA approved

File: src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewController.swift (hunk 1 above)
Comment (pascal-brause) [reply src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewController.swift:18]: yeah, good point. I will change it.

**Pull request merged**